blocksuite/affine/block-database/src/data-source.ts

```diff
@@ -306,8 +318,13 @@ export class DatabaseBlockDataSource extends DataSourceBase {
       type: toType,
       data: result.property,
     }));
+    const metaConfig = this.propertyMetaGet(toType);
     const cells: Record<string, unknown> = {};
     currentCells.forEach((value, i) => {
+      if (value == null && result.cells[i] == null) {
+        cells[rows[i]] = metaConfig.config.defaultValue;
+      }
```

Comment:
it updates cells to default value whenever the property is updated to other type.